piksi_tools/console/console.py

```diff
@@ -221,15 +233,17 @@ class SwiftConsole(HasTraits):
     title = CONSOLE_TITLE
   )
 
+
   def print_message_callback(self, sbp_msg, **metadata):
     try:
-      self.console_output.write(sbp_msg.payload.encode('ascii', 'ignore'))
+      self.console_output.write_level(sbp_msg.payload.encode('ascii'),
+                                      str_to_log_level(msg.split(':')[0]))
     except UnicodeDecodeError:
       print "Critical Error encoding the serial stream as ascii."
 
   def log_message_callback(self, sbp_msg, **metadata):
     try:
-      self.console_output.write(MsgLog(sbp_msg).text.encode('ascii', 'ignore'))
+      self.console_output.write_level(MsgLog(sbp_msg).text.encode('ascii', 'ignore'), MsgLog(sbp_msg).level)
```

Comment:
The item wasn't a MsgLog, it was just a message.  I'll try without the cast and see if it works today.